apps/fabric-website/src/components/Nav/Nav.CollapsibleSectionTitle.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+import { createComponent } from '@uifabric/experiments/lib/Foundation';
+import { CollapsibleSectionTitleView as view } from '@uifabric/experiments/lib/components/CollapsibleSection';
+import { getStyles as styles } from './Nav.CollapsibleSectionTitleStyles';
+import {
+  ICollapsibleSectionTitleProps,
+  ICollapsibleSectionTitleStyles
+} from '@uifabric/experiments/src/CollapsibleSection';
```

Comment:
I think you should aim to import just from `@uifabric/experiments`. Same with the import above